ntbs-service/Pages/Shared/_Header.cshtml

```diff
@@ -57,7 +57,7 @@
     <div id="header-nav" class="header-navigation-background">
         <div class="govuk-header__container govuk-width-container no-border">
             <nav>
-                <ul id="navigation" class="govuk-header__navigation header-navigation-list" aria-label="Top Level Navigation">
+                <ul id="navigation" class="govuk-header__navigation govuk-header__navigation-list header-navigation-list" aria-label="Top Level Navigation">
```

Comment:
This looks good. Have you done regression testing to ensure that this hasn't had any negative side-effects? In particular, the focus of NTBS-359 was notification print, so we should definitely regression test that.

Reply:
No, this sounds good :-)